examples/java-matter-controller/java/src/com/matter/controller/commands/common/MatterCommand.java

```diff
@@ -113,4 +118,35 @@ private void maybeSetUpStack() throws Exception {
   private void maybeTearDownStack() {
     // ToDo:We need to call DeviceController::Shutdown()
   }
+
+  public void setTestResult(String result) {
+    TestResult = result;
+  }
+
+  public void expectSuccess(long timeout) {
+    expectResult("Success", timeout);
+  }
+
+  private void expectResult(String expectedResult, long timeout) {
+    long start = System.currentTimeMillis();
+    while (TestResult == null)
+      try {
+        if (System.currentTimeMillis() > (start + timeout)) {
+          throw new RuntimeException("timeout!");
+        }
+        Thread.sleep(100);
+      } catch (Exception ex) {
+      }
+
+    if (!TestResult.equals(expectedResult)) {
+      if (!expectedResult.equals("Success")) {
+        System.out.format(
+            "%s command failed:%n    Expected: %s%n    Got: %s%n",
+            getName(), expectedResult, TestResult);
+        throw new RuntimeException(getName());
+      } else {
+        System.out.format("%s command failed: %s%n", getName(), TestResult);
+      }
```

Comment:
We should clear mTestResult after usage, otherwise, the next test will always go through

Reply:
done, thanks